scripts/pythondistdeps.py

```diff
@@ -271,6 +278,19 @@ def get_marker_env(dist, extra):
     # At least one type of normalization must be provided
     assert normalized_names_provide_pep503 or normalized_names_provide_legacy
 
+    # Is this script being run for an extras subpackage?
+    extras_subpackage = None
+    if args.package_name and '+' in args.package_name:
+        # The extras names are encoded in the package names after the + sign.
+        # We take the part after the rightmost +, ignoring when empty,
+        # this allows packages like nicotine+ or c++ to work fine.
+        # While packages with names like +spam or foo+bar would break,
+        # names started with the plus sign are not very common
+        # and pluses in the middle can be easily replaced with dashes.
+        # Python extras names don't contain pluses according to PEP 508.
+        package_name_parts = args.package_name.rpartition('+')
+        extras_subpackage = package_name_parts[2] or None
```

Comment:
This seem to include https://src.fedoraproject.org/rpms/python-rpm-generators/c/7398b71fbceeabd090c8d768edbf75e9ecaa60af as well, good.

Reply:
Yes, forgot to mention that commit.